supreme_court_predictions/processing/datacleaner.py

```diff
@@ -23,16 +23,13 @@ class DataCleaner:
     it into a usable format.
     """
 
-    def __init__(self, downloaded_corpus, save_data=True):
+    def __init__(self):
         self.cases_df = None
         self.clean_case_ids = None  # stores the case IDs to use
         self.clean_utterances_list = None
         self.speakers_df = None
         self.utterances_df = None
 
-        self.downloaded_corpus = downloaded_corpus
-        self.save_data = save_data
```

Comment:
I can't think of a reason to not save the data. This step is also called after `get-data`, so the corpus flag is redundant.